plotly/tests/test_optional/test_figure_factory.py

```diff
@@ -422,7 +469,7 @@ def test_dendrogram_random_matrix(self):
         self.assert_dict_equal(dendro['layout'], expected_dendro['layout'])
 
     def test_dendrogram_orientation(self):
-        X = np.random.rand(5, 5) 
+        X = np.random.rand(5, 5)
```

Comment:
Your IDE probably did this automatically on save or something, but it's helpful if you *don't* introduce changes to lines of code which are unrelated to the problem you're addressing. Not a huge deal, but something to be mindful of.